src/diet.ts

```diff
@@ -715,6 +757,12 @@ export function consumeDiet(diet: Diet<Note>, name: DietName): void {
           synthesize(countToConsume, $effect`Synthesis: Greed`);
         } else if (getClanLounge()[`${menuItem.item}`] && itemType(menuItem.item) === "booze") {
           cliExecute(`drink ${menuItem.item}`);
+        } else if (menuItem.item === $item`campfire hot dog`) {
```

Comment:
this is getting to be a big enough statement that we might want to move to switch or something

Reply:
maybe a map?